src/global/translations.ts

```diff
@@ -21,7 +21,11 @@ const allTranslations = {
 const REGEX = /\{\s*(\w+)\s*\}/g;
 
 export class Translations {
-    public get(key: string, language = 'en', params?: object): string {
+    public get(
+        key: string,
+        language = 'en',
+        params?: object,
+    ): string | undefined {
```

Comment:
@jgroth I rebased on main and there was a change in `translations.st` file which was conflicting with my change. This is how the conflict looked like:

![image](https://github.com/user-attachments/assets/fbab4a4d-2396-4084-862b-2f297e6d9842)

The `string` type was already added in this PR
https://github.com/Lundalogik/lime-elements/pull/3464/files#diff-8a5b1230d00b72dede2900072c8f3d7fae328c9488f7ed5aba33b6f3d19385f4

So I just added the `| undefined`.

You need to reapprove this PR

Reply:
But the new implementation can no longer return `undefined` so this change is no longer needed :smile:

Reply:
⚡